integrations/gitlab/gitlab_integration/core/paging.py

```diff
@@ -31,16 +31,7 @@ async def fetch(
             ],
         ],
         validation_func: Callable[
-            [
-                Union[
-                    RESTObject,
-                    ProjectPipelineJob,
-                    Issue,
-                    ProjectPipeline,
-                    Dict[str, Any],
-                    Project,
-                ]
-            ],
+            [Any],
```

Comment:
Can you open a task on me to fix this typing?

Reply:
I'm pretty sure this is the correct typing though.

The paging shouldn't care what the validation function it gets; Each validation function should define whatever parameters it wants, Shouldn't it?